packages/js/client/src/__tests__/e2e/wasm-rs.spec.ts

```diff
@@ -269,6 +269,11 @@ describe("wasm-rs test cases", () => {
       wrapperUri
     );
   });
+
+  it("codegen-manifest", async () => {
```

Comment:
Yes, exactly. It wouldn't build if it wasn't working. Instead of generating code to the "./src/wrap" folder, it's generating code in './src/wrapx' and reading from there.

Reply:
IDK if this would impact anything deeper, but a nice little piece of syntactic sugar could be that we explicitly say "Hey, we expect that this won't throw.
Something like:
```typescript
expect(async () => {
    const wrapperPath = `${GetPathToTestWrappers()}/wasm-rs/codegen-manifest`;
    await buildWrapper(wrapperPath);
}).not.toThrow();
```
Total nitpick, we're just making sure that it's immediately obvious from the code what we're expecting.